lib/urlwatch/jobs.py

```diff
@@ -254,7 +254,7 @@ def retrieve(self, job_state):
         file_scheme = 'file://'
         if self.url.startswith(file_scheme):
             logger.info('Using local filesystem (%s URI scheme)', file_scheme)
-            return open(self.url[len(file_scheme):], 'rt').read()
+            return (open(self.url[len(file_scheme):], 'rt').read(), 'text/plain')
```

Comment:
The local file doesn't necessarily need to be `text/plain`, could be a different type?

Reply:
Could be configured manually per job. And if not defined per job, then fallback could be taken from settings. If no fallback in settings, then libmagic can be used.

Reply:
Yes, make it an `__optional__` setting in the Job (it make sense for file jobs), maybe `mimetype` or something and if not set, use `text/plain`.